### PR TITLE
Instance Feature & `isReactComponent` Update

### PR DESCRIPTION
**Pull request by jcmcneal**
#23  
#25 
#27 
#28

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -144,7 +147,9 @@ export default class StepWizard extends PureComponent {
     }
 
     // Allows for using HTML elements as a step
-    isReactComponent = type => typeof type === 'function'
+    isReactComponent = ({ type }) => (
+        typeof type === 'function' || typeof type === 'object'
```
Comment (jcmcneal) [inline src/index.js]: @mathvaleriano I think this should work for you, yeah?

Comment (mathvaleriano) [reply src/index.js]: yeah... i'm using it for now

Comment (mathvaleriano) [reply src/index.js]: https://github.com/jcmcneal/react-step-wizard/issues/27#issuecomment-486482549

File: src/index.js [hunk 2]
```diff
@@ -179,7 +184,7 @@ export default class StepWizard extends PureComponent {
         });
 
         return (
-            <div>
+            <div className={this.props.className}>
```
Comment (jcmcneal) [inline src/index.js]: #30 This will allow you to pass `className` to `StepWizard`

**Pull request merged**